<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_line_next</class>
 <widget class="QDialog" name="add_line_next">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加通路</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>添加通路</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>通路端点1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbx_add_num1">
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>通路端点2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbx_add_num2">
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QRadioButton" name="rbtn_add_dbl">
               <property name="text">
                <string>双向</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbtn_add_sgl">
               <property name="text">
                <string>单向</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="pbtn_add_ok">
              <property name="text">
               <string>确认添加</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbtn_add_can">
              <property name="text">
               <string>取消</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>修改通路</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>通路编号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbx_chg_rdnum">
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>通路端点1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbx_chg_stnum1">
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>通路端点2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbx_chg_stnum2">
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QRadioButton" name="rbtn_chg_dbl">
               <property name="text">
                <string>双向</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbtn_chg_sgl">
               <property name="text">
                <string>单向</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QPushButton" name="pbtn_chg_ok">
              <property name="text">
               <string>确认修改</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbtn_chg_can">
              <property name="text">
               <string>取消</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>删除道路</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,1">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>通路编号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbx_del_num">
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QPushButton" name="pbtn_del_ok">
              <property name="text">
               <string>确认删除</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbtn_del_can">
              <property name="text">
               <string>取消</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>通路情况</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QTextBrowser" name="txb_rd"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="title">
        <string>站点情况</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextBrowser" name="txb_st"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QPushButton" name="pbtn_back">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_clr">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
